src/plugins/intel_cpu/src/transformations/cpu_opset/common/pass/convert_matmul_to_fc.cpp

```diff
@@ -29,6 +30,19 @@ ov::intel_cpu::ConvertMatMulToFC::ConvertMatMulToFC() {
         // So in case of adding new operations that takes matmul inputs we need keep update fc_input_a and fc_input_b.
         auto fc_input_a = pattern_map.at(activations_m);
         auto fc_input_b = pattern_map.at(weights_m);
+        bool is_convert = false, is_constant = false;
+        if (auto convert_node = std::dynamic_pointer_cast<ngraph::opset1::Convert>(fc_input_b.get_node_shared_ptr())) {
+            if (convert_node->get_input_size() == 1 && convert_node->get_output_size() == 1 &&
```

Comment:
this should be always true `convert_node->get_input_size() == 1 && convert_node->get_output_size() == 1`

Reply:
removed, thanks